src/Definition.php

```diff
@@ -89,4 +89,11 @@ class Definition
      * @var string
      */
     public $documentation;
+
+    /**
+     * Parameters array (for methods and functions), for use in textDocument/signatureHelp
+     *
+     * @var string[]
```

Comment:
It is already done, and the docs are in place as well. I added a comment for you though - `Param` nodes do not have a `parentNode` attribute - I could not fix it - the attribute was added in `ReferenceAdder` but then was not available at a later point.